File: 2020/rust/template.rs
fn part1(input: &String) -> i32 {
    input.chars().count() as i32
}

fn part2(input: &String) -> i32 {
    input.lines().count() as i32
}

fn main() -> std::io::Result<()> {
    let input: String = std::fs::read_to_string("../inputs/01")?;

    println!("Part 1: {}", part1(&input));
    println!("Part 2: {}", part2(&input));

    Ok(())
}

File: 2025/typescript/template.ts
import * as fs from 'fs';
import * as path from 'path';

export function part1(input: string): number {
  const lines = input.trim().split('\n');
  // TODO: Implement solution
  return 0;
}

export function part2(input: string): number {
  const lines = input.trim().split('\n');
  // TODO: Implement solution
  return 0;
}

// Run if executed directly
if (require.main === module) {
  const input = fs.readFileSync(path.join(__dirname, '../inputs/XX.txt'), 'utf-8');
  console.log('Part 1:', part1(input));
  console.log('Part 2:', part2(input));
}
